### ISSUE TITLE
Test run fails with WHERE ApexClassorTriggerId IN () ERROR at Row:1:Column:162 unexpected token: ')' 14/01 relase

### ISSUE THREAD

**Issue by PawelWozniak** [forcedotcom/cli#808]
### Summary

Sometimes when I run a test that fails I am getting an error in CLI, test class is run on SF and results can be checked in SF UI:
16:05:57.228 Starting Run Apex Tests

```
WHERE ApexClassorTriggerId IN ()
                              ^
ERROR at Row:1:Column:162
unexpected token: ')'
```

### Steps To Reproduce:

1 Run test class that fails. It don't happen always. Rather after longer time of running tests. 

### Expected result

I should see test run results.

### Actual result

I see an error. Probably internal CLI error as test don't fail with such message.

### Additional information

**SFDX CLI Version**(to find the version of the CLI engine run sfdx --version): sfdx-cli/7.84.2-a2868a68d5 win32-x64 node-v12.18.3
**OS and version**: Windows 10

**Comment by SubC4i** [comment]
I was experiencing the exact same issue as you (just now) and received the same exact error message in VSCode.  I looked into the "Apex Test Execution" section under Setup and found that one of my test records created within the testSetup method was hitting a validation rule.  Viewing the "Apex Test Result Detail" revealed the actual validation rule message and the line within the testSetup that hit the rule.  My guess is that the CLI doesn't know what to do when the error is hit within the testSetup method and a validation rule hits.  May not be for the same reason for you, but this lined up for me.

**Comment by nabondance** [comment]
Same error here too.
The test fail in VSCode but works perfectly in Salesforce developer console.
It does it for every tests even the more basic ones (no setup, no assert, nothing).

EDIT:
After more investigating, the cli is working in command line. The error seams to appears only in the integrated VSCode testing feature.

**Comment by maaaaarco** [comment]
This is an issue related to the VSCode Extension. 
In this discussion there's a workaround to disable a beta feature https://github.com/forcedotcom/salesforcedx-vscode/issues/2863
;)

**Comment by AnanyaJha** [comment]
As Marco said, this is an issue with the latest release of the VS Code Extensions - this is part of a beta feature that can be disabled by going into preferences and turning off the `Salesforcedx-vscode-core › Experimental: Use Apex Library` setting. We're working on a fix and will update this thread when it's available, thanks!

**Comment by AnanyaJha** [comment]
This issue was addressed with the latest release of the VS Code Extensions. Please update to version `50.14.0` and turn the setting back on to see the changes. Let us know if you have any other feedback, thanks!

**Comment by WillieRuemmele** [comment]
Closing this issue as it seems to be resolved in `50.14.0` of the VS Code Extensions
